Composer/packages/client/src/pages/diagnostics/Diagnostics.tsx

```diff
@@ -3,23 +3,33 @@
 
 /** @jsx jsx */
 import { jsx } from '@emotion/core';
-import { useState } from 'react';
-import { RouteComponentProps } from '@reach/router';
+import { Fragment, Suspense, useEffect, useState } from 'react';
+import { RouteComponentProps, Router } from '@reach/router';
+import formatMessage from 'format-message';
+import { useRecoilValue } from 'recoil';
 
-import { Toolbar } from '../../components/Toolbar';
 import { navigateTo } from '../../utils/navigation';
 import { convertPathToUrl } from '../../utils/navigation';
+import { Page } from '../../components/Page';
+import { LoadingSpinner } from '../../components/LoadingSpinner';
+import { diagnosticNavLinksSelector } from '../../recoilModel/selectors/diagnosticsPageSelector';
+import { currentProjectIdState } from '../../recoilModel';
 
-import useNotifications from './useNotifications';
-import { NotificationList } from './NotificationList';
-import { NotificationHeader } from './NotificationHeader';
-import { root } from './styles';
+import { NotificationList } from './DiagnosticList';
+import { NotificationFilter } from './DiagnosticFilter';
 import { INotification, NotificationType } from './types';
 
-const Notifications: React.FC<RouteComponentProps<{ projectId: string }>> = (props) => {
-  const { projectId = '' } = props;
-  const [filter, setFilter] = useState('');
-  const notifications = useNotifications(projectId, filter);
+const Notifications: React.FC<RouteComponentProps<{}>> = (props) => {
```

Comment:
Component needs to be renamed to Diagnostics and RouteComponentProps\<{projectId; skillId}